Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG ARCH=
-FROM ${ARCH}golang:1.16 AS builder
+FROM ${ARCH}golang:1.17 AS builder
 WORKDIR /go/src/github.com/superorbital/cludo/
-RUN go get github.com/ahmetb/govvv
```

Comment:
I'm not a gopher, so this may be dumb, but it seems fragile to install `@latest` for these build tools.  Would it make sense to use a second `go.mod` file for them (like `go.development.mod` or something)?  If so, I can create a new issue for that.

Reply:
It's a bit of a tradeoff. These are so stable that this helps ensure that we get small bug and security fixes as they show up, but does mean that builds are a little less reproducible, if there was a functional change that crept in. I could go either way in this case, since we are primarily using these as CLI tools and not as actual code libraries.

Reply:
It also looks like `govvv` needs to be latest at the moment, as a few bug fixes are still not is the latest release.

Reply:
I did a bit of research and found the preferred approach for this and have update things accordingly.